docs/guides/add-validator-with-curl.md

```diff
@@ -42,8 +43,8 @@ Further below, you will find instructions about how to get these information by
   Keep it safe and secure!
 - **Wallet address & private key**: You will need your wallet address and private key.<br/>
   Remember to keep your private key safe!
-- **100,000 unlocked CAMs** in your P-Chain wallet.<br/>
-  You can check this on the online [wallet](https://wallet.camino.network) to ensure that you have at least 100,000 unlocked tokens on the P-Chain.
+- **100,000 unbonded CAMs** in your P-Chain wallet. And a few CAMs for paying for transaction fees.<br/>
```

Comment:
I'd say not-bonded, but may be I'm wrong :)

Reply:
I've checked a few dictionaries and settled for **non-bonded** :)